src/lighteval/metrics/metrics.py

```diff
@@ -550,6 +558,14 @@ class Metrics(Enum):
         corpus_level_fn=CorpusLevelPerplexityMetric("perplexity").compute,
         higher_is_better=False,
     )
+    target_perplexity_loglikelihood_acc = SampleLevelMetric(
```

Comment:
Not what this metric means ? How do we mic ppl and acc ?

Reply:
should be renamed to `loglikelihood_acc_multicontext` I think - I was initially going to reuse the perplexity but it ended up being easier to change the context management.
Btw, would we want to add multiple context by default?

Reply:
you mean not having to add it in `specifics` of the task ?

Reply:
yep! for ex - `context: str|list[str]`

Reply:
I think having context: str | list[str] would be the cleanest solution 👍.
Altough it will require a lot of branching, so maybe nicer would be to use context: list[str] interanaly and broadcast when needed

Reply:
Almost all our use cases will be  `context: str` so I'd rather do some handling behind the scenes instead (to go to a list) and allow to create a Doc with a str context (will be confusing for most users otherwise)

Reply:
👍 Yeah make sense. Also this approach will be way better as we don't need new metrics and we can reuse the old one